pom.xml

```diff
@@ -23,11 +23,11 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <protobuf.version>4.28.2</protobuf.version>
-        <grpc.version>1.65.0</grpc.version>
-        <netty.version>4.1.111.Final</netty.version> <!-- match version grpc-netty depends on -->
+        <protobuf.version>4.31.0</protobuf.version>
```

Comment:
This looks like an increase in the protobuf version from even 4.28.2. I am assuming behavior will be the same as 4.28? Thanks